scripts/showSOM.py

```diff
@@ -9,24 +9,7 @@
 import math
 import getopt
 import sys
-
-def print_usage():
-    print ('')
-    print ('Usage:')
-    print ('')
-    print ('  showSOM.py [Options] <inputfile>')
-    print ('')
-    print ('Options:')
-    print ('')
-    print ('  --help, -h                   Print this lines.')
-    print ('  --save, -s <String>          Location to save JPGs to.')
```

Comment:
SOM loction path can be set with SOM name. SOM name should not be optional, but a positional parameter, since a SOM file will always be written.

Reply:
I changed my mind, after I noticed that  list of files will be written. Therefore, it's fine for me. Maybe the option can renamed as 'location' instead of 'save'.